src/buildifier/buildifier_availability.ts

```diff
@@ -33,9 +35,15 @@ const BUILDTOOLS_RELEASES_URL =
  */
 export async function checkBuildifierIsAvailable() {
   const buildifierExecutable = getDefaultBuildifierExecutablePath();
+
   // Check if the program exists (in case it's an actual executable and not
   // an target name starting with `@`).
-  if (!buildifierExecutable.startsWith("@")) {
+  const isTarget = buildifierExecutable.startsWith("@")
+
+  // Check if the program exists as a relative path of the workspace
+  const isRelative = fs.existsSync(path.join(vscode.workspace.workspaceFolders?.[0]?.uri?.fsPath, buildifierExecutable));
```

Comment:
What happens for `@` paths here? `@` is not a valid character in file system paths. Does `fs.exists` gracefully return `false` for such file paths or are we getting some kind of exception?

(Maybe just move the complete `fs.existsSync` check into the `if(!isTarget)`, just to be on the safe side)

Reply:
Since we'll do a check if the file exists and it isn't a valid path name it'll just set the `isRelative` (now `pathExists`) to false